app/web/src/components/NoSelectionDetailsPanel.vue

```diff
@@ -57,13 +58,43 @@ import {
   PillCounter,
 } from "@si/vue-lib/design-system";
 import clsx from "clsx";
+import { computed } from "vue";
 import ApplyChangeSetButton from "@/components/ApplyChangeSetButton.vue";
 import { useChangeSetsStore } from "@/store/change_sets.store";
 import { useActionsStore } from "@/store/actions.store";
+import { useComponentsStore } from "@/store/components.store";
 import SidebarSubpanelTitle from "./SidebarSubpanelTitle.vue";
 import ChangesPanel from "./ChangesPanel.vue";
 import SecretsPanel from "./SecretsPanel.vue";
 
+const componentsStore = useComponentsStore();
 const changeSetStore = useChangeSetsStore();
 const actionsStore = useActionsStore();
+
+const diffs = computed(() => {
```

Comment:
We should remove diffs from this calculation - they are no longer on the panel. I removed them in #3675

Reply:
ah okay! will do

Reply:
done!